gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/QETopBar.kt

```diff
@@ -16,16 +17,45 @@ import androidx.compose.material3.TextButton
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalUriHandler
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import com.gravatar.GravatarConstants
 import com.gravatar.quickeditor.R
 import com.gravatar.ui.GravatarTheme
 
 @Composable
-internal fun QETopBar(onDoneClick: () -> Unit, modifier: Modifier = Modifier, onGravatarIconClick: () -> Unit = {}) {
+internal fun QETopBarWithContent(
+    onLeftButtonClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    leftButtonLabel: String = stringResource(R.string.gravatar_qe_bottom_sheet_done),
+    content: @Composable () -> Unit = {},
+) {
```

Comment:
❓ I'm not sure the slot API is a good choice here. This serves well when you assign certain spaces for item in the TopBar, but the content here is not the scope of the TopBar at all. Any reasons you went with this? 

I imagine that it's likely to avoid doing this
```Column { TopBar ...```

but we could achieve this by having a `QEPage` component that would make this, rather than making the TopBar aware of the content at all.

Reply:
I've just pushed some changes related to this. Let me know what you think.